templates/checkpoint_gaia_show_arp_dynamic_all.textfsm

```diff
@@ -0,0 +1,8 @@
+Value ADDRESS (\d+\.\d+\.\d+\.\d+)
+Value MAC (\S+)
+
+Start
+  ^Dynamic\sArp\sParameters
+  ^IP\sAddress\s+Mac\sAddress
+  ^${ADDRESS}\s+${MAC} -> Record
+  ^. -> Error
```

Comment:
Looks great! If you don't mind changing the following and then we get this merged.
```suggestion
 ^\s*$$
 ^. -> Error
```